app/pb_kits/playbook/pb_list/_list.jsx

```diff
@@ -19,6 +19,7 @@ type ListProps = {
   tabIndex?: String,
   size?: String,
   xpadding: Boolean,
+  checkbox_list: Boolean,
```

Comment:
For any JSX files, you will want to use camelCase `checkboxList` like so. 

```suggestion
  checkboxList: Boolean,
```